src/testsuite/arith128_print.c

```diff
@@ -976,6 +999,38 @@ check_v2b64x_priv (char *prefix, vb64_t val128, vb64_t shouldbe)
   return (rc);
 }
 
+int
+check_vb128c_priv (char *prefix, vb128_t val128, vb128_t shouldbe)
+{
+  int rc = 0;
+
+  if (vec_any_ne((vui32_t ) val128, (vui32_t ) shouldbe))
```

Comment:
nit: extra whitespace within casts.
global comment (appears below as well, and elsewhere)